src/workers/parser/tests/fixtures/functionNames.js

```diff
@@ -38,3 +38,5 @@ class Cls {
 (function(){});
 
 export default function (){}
+
+function def(first = defaultValue, second){}
```

Comment:
we didn't get any values because `function(first = defaultValue, second){}` is a syntax error